includes/views/handlers/message_handler_field_message_render.inc

```diff
@@ -12,6 +12,9 @@
  */
 class message_handler_field_message_render extends views_handler_field {
 
+  // Message entities
+  var $messages = array();
```

Comment:
I would avoid this `$messages` - as we end up caching this inside `message_load` and here. Instead - you can do something like:

```php
function pre_render() {

...
+    if (empty($mids)) {
        // No Message IDs.
        return;
     }
     
    // Load the Messages in a single call, so they will be statically cached in the ::render function.
     message_load_multiple($mids);
}
```

Then, the `render` function will stay as-is, the `message_load` will however fetch the `$message` from the static cache.